### DISCUSSION TITLE
Utilite suplementaire pour les pet

### DISCUSSION THREAD

**Discussion by Archimageifrit** [Crownicles/Crownicles#1108] [Nouvelles idées]
Mon idée serait de donner une utilité en plus au pet en Les permettant de nous aidé pendant les combats  , sa pourrait se faire de différentes manières ( à voir laquelle et la meilleure )

-   soit avec un buf qui dépendra du pet que l'on a ( exemple boost de force avec un loup ) 
-  Une attaque "pet"   ou c'est le pet qui attaque à notre place , avec différent effect selon le pet , genre une simple attaque avec un chien , provoque pour le coq ( qui prendra une parti des dégât pour l'exemple ) ou un heal pour certain pet comme la licorne ) et les attaques seront meilleurs selon les ⭐stars du pet  ( on peu également supposé que forcé un pet à se battre puisse baisser son moral en contrepartie )
-  Que les pet face des sorte d'attaque combinés avec nous , par exemple je fait une attaque rapide et mon pet en profite pour infligé 2 , 3 dégât à mon opposant )

**Comment by BastLast** [comment]
déjà prévu aha (pas exactement tel que décrit dans cette suggestion mais yaura du nouveau là dessus :) )

**Reply by Archimageifrit** [reply]
Yes , après je m'en doute un peu que  c'est pas comme je l'écris telle qu'elle , c'est pour ça que je met des idée plutôt différente pour que les deg prennent ce qui les arrange , mais si c'est déjà prévu c'est génial
